plugins/push/frontend/public/javascripts/countly.views.js

```diff
@@ -1,20 +1,20 @@
-/* global countlyVue,app,CV,countlyPushNotification,CountlyHelpers,jQuery,countlyManagementView,countlyCommon,$,countlyGlobal,countlyAuth,countlySegmentation,countlyUserdata, components,Backbone*/
+/* global countlyVue,app,CV,countlyPushNotification,CountlyHelpers,jQuery,countlyManagementView,countlyCommon,$,countlyGlobal,countlyAuth,countlySegmentation,countlyUserdata,components,Backbone,moment*/
 (function() {
 
     var AUTOMATIC_PUSH_NOTIFICATION_STATUS_FILTER_OPTIONS = [
-        {label: CV.i18n("push-notification.table-filter-scheduled"), value: countlyPushNotification.service.StatusEnum.SCHEDULED},
-        {label: CV.i18n("push-notification.table-filter-all"), value: countlyPushNotification.service.StatusEnum.ALL},
-        {label: CV.i18n("push-notification.table-filter-sent"), value: countlyPushNotification.service.StatusEnum.SENT},
-        {label: CV.i18n("push-notification.table-filter-sending"), value: countlyPushNotification.service.StatusEnum.SENDING},
-        {label: CV.i18n("push-notification.table-filter-aborted"), value: countlyPushNotification.service.StatusEnum.ABORTED},
-        {label: CV.i18n("push-notification.table-filter-failed"), value: countlyPushNotification.service.StatusEnum.FAILED},
-        {label: CV.i18n("push-notification.table-filter-stopped"), value: countlyPushNotification.service.StatusEnum.STOPPED}
+        {label: CV.i18n("push-notification.status-scheduled"), value: countlyPushNotification.service.StatusEnum.SCHEDULED},
+        {label: CV.i18n("push-notification.status-all"), value: countlyPushNotification.service.StatusEnum.ALL},
+        {label: CV.i18n("push-notification.status-sent"), value: countlyPushNotification.service.StatusEnum.SENT},
+        {label: CV.i18n("push-notification.status-sending"), value: countlyPushNotification.service.StatusEnum.SENDING},
+        {label: CV.i18n("push-notification.status-aborted"), value: countlyPushNotification.service.StatusEnum.ABORTED},
+        {label: CV.i18n("push-notification.status-failed"), value: countlyPushNotification.service.StatusEnum.FAILED},
+        {label: CV.i18n("push-notification.status-stopped"), value: countlyPushNotification.service.StatusEnum.STOPPED}
```

Comment:
Just to reduce redundant translations I suppose

Reply:
correct.